data/core/macros/abilities.cfg

```diff
@@ -186,6 +186,13 @@ This ability will not cure an affected unit of poison, however, only delay its e
                 [/filter_adjacent]
             [/filter]
         [/affect_adjacent]
+        [event]
+            name=preload,unit placed
```

Comment:
Does this affect events that place a unit and also have `[allow_undo]`?

Reply:
Good point. I think we can just add `[allow_undo]` to these events, probably? Making it actually remove the events if they were added probably isn't possible though… since we can't tell the difference between "they were added" and "they already existed so they weren't added".

Reply:
Is it better to add `[allow_undo]` or just remove `unit placed` and let `preload` do the work by itself?

Also, should I add `first_time_only=no`?

Reply:
Okay, added the `[allow_undo]` tag to them.